projects/power_distribution/inc/bps_watcher.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include "can.h"
+#include "can_msg_defs.h"
+#include "can_transmit.h"
+#include "can_unpack.h"
+
+// This module simply watches for the BPS_HEARTBEAT
+// to make sure that no fault has occured
```

Comment:
Please add the `// Requires X, Y, and Z to be initialized` comment for easier debugging if something's not initialized